plugin/hover.py

```diff
@@ -65,7 +65,11 @@ def run(self, edit: sublime.Edit, point: Optional[int] = None, event: Optional[d
         window = self.view.window()
         if not window:
             return
-        self._base_dir = windows.lookup(window).get_project_path(self.view.file_name() or "")
+        wm = windows.lookup(window)
+        listener = wm.listener_for_view(self.view)
+        if not listener:
+            return
```

Comment:
Would checking if there exists a `hoverProvider` session be even better?

Reply:
Maybe not since there is some code that fetches code actions for diagnostics. But then maybe just checking whether any session is running?

Reply:
Your question inspired me to tackle the problem in a better way. See 23bd979